cypress/intercepts/admin.ts

```diff
@@ -0,0 +1,22 @@
+const createAdmin = () => {
+  cy.intercept("POST", "/accounts", {
+    statusCode: 200,
+    body: {
+      id_account: 1,
+      email: "",
+      account_category: "admin",
+    },
+  });
```

Comment:
```suggestion
  }).as("createAdmin");
```

This is so you test and verify if backend call was done